web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/GetSickLeaveSummaryServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.rehabstod.service.sjukfall;
+
+import java.util.ArrayList;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.rehabstod.service.pu.PuService;
+import se.inera.intyg.rehabstod.service.sjukfall.dto.SjukfallSummary;
+import se.inera.intyg.rehabstod.service.sjukfall.statistics.StatisticsCalculator;
+import se.inera.intyg.rehabstod.web.controller.api.dto.SickLeavesFilterRequestDTO;
+
+@Service
+public class GetSickLeaveSummaryServiceImpl implements GetSickLeaveSummaryService {
+
+    private final PuService puService;
+    private final StatisticsCalculator statisticsCalculator;
+    private final GetActiveSickLeavesService getActiveSickLeavesService;
+
+    @Autowired
```

Comment:
Correct me if im wrong, but I think we are using a spring version later than 4.3? 

As of version 4,3 constructors don't need to be annotated with @Autowired, since its handled by spring.


>As of Spring Framework 4.3, an @Autowired annotation on such a constructor is no longer necessary if the target bean only defines one constructor to begin with. However, if several constructors are available, at least one must be annotated to teach the container which one to use.